### ISSUE TITLE
Issue with non-NFC devices

### ISSUE THREAD

**Issue by selmaskopljakovic** [okadan/flutter-nfc-manager#28]
I made an android app that has NFC scanning and it works perfectly. For non-NFC devices, I made another option for scanning. The problem is that I can't even build an app on a non-NFC device. There is an exception:

```
Caused by: java.lang.IllegalStateException: NfcAdapter.getDefaultAda…nding.applicationContext) must not be null
E/AndroidRuntime(30997): 	at io.flutter.plugins.nfcmanager.NfcManagerPlugin.onAttachedToEngine(NfcManagerPlugin.kt:65)
E/AndroidRuntime(30997): 	at io.flutter.embedding.engine.FlutterEnginePluginReg

```

What to do to have buildable app for any android device but to use NFC with nfc-manager on device with NFC?
Thanks

**Comment by okadan** [comment]
Thank you for reporting.

I think the plugin needs to be fixed.

I will publish a new version on pub.dev soon.

**Comment by okadan** [comment]
@selmaskopljakovic 

Fixed and published v2.0.1 to pub.dev.

Commit: https://github.com/okadan/flutter-nfc-manager/commit/61e9af15d215a36f22ec8956eb1897e3d223b02d

**Comment by selmaskopljakovic** [comment]
@okadan 
I just checked and it works :). 
Great!
Thank you
